base/essentials.jl

```diff
@@ -1042,6 +1042,7 @@ call obsolete versions of a function `f`.
     Prior to Julia 1.9, this function was not exported, and was called as `Base.invokelatest`.
 """
 function invokelatest(@nospecialize(f), @nospecialize args...; kwargs...)
+    @inline
```

Comment:
Just noticing that codegen was doing dumb stuff here because of the nospecialize. It would generate a call to make a Tuple for the args so that it could call the apply function to unpack those again to call the builtin 🙄